controller/bin/boot

```diff
@@ -65,6 +65,7 @@ etcd_safe_mkdir /deis/services
 echo "controller: running etcd data migrations..."
 for script in /app/migrations/data/*.sh;
 do
+	# shellcheck disable=SC1090
```

Comment:
Nifty! I didn't know that this was possible with shellcheck (disabling a shellcheck check with a bash comment).